packages/larva-patterns/modules/vlanding-video-showcase/vlanding-video-showcase.prototype.js

```diff
@@ -2,24 +2,34 @@ const clonedeep = require( 'lodash.clonedeep' );
 
 const vlanding_video_card = clonedeep( require( '../../modules/vlanding-video-card/vlanding-video-card.prototype' ) );
 const social_share = clonedeep( require( '../../modules/social-share/social-share.prototype' ) );
+const c_span = clonedeep( require( '../../components/c-span/c-span.prototype' ) );
 const c_title = clonedeep( require( '../../components/c-title/c-title.prototype' ) );
 const c_tagline = clonedeep( require( '../../components/c-tagline/c-tagline.prototype' ) );
 const c_heading = clonedeep( require( '../../components/c-heading/c-heading.prototype' ) );
+
 const vlanding_video_showcase_card = clonedeep( vlanding_video_card );
 vlanding_video_card.c_title = false;
 vlanding_video_card.c_span = false;
 
+vlanding_video_showcase_card.c_title.c_title_url = false;
+vlanding_video_showcase_card.c_title.c_title_classes = 'lrv-u-color-black lrv-u-font-size-14 lrv-u-font-weight-normal lrv-u-padding-t-050';
+vlanding_video_showcase_card.c_title.c_title_link_classes = 'lrv-u-color-black';
+
+vlanding_video_showcase_card.c_span.c_span_classes += ' lrv-u-display-block lrv-u-padding-b-00 lrv-u-padding-t-050 lrv-u-font-size-12 lrv-u-font-weight-normal';
+
 c_title.c_title_classes = 'lrv-u-font-size-24 lrv-u-font-size-32@desktop lrv-u-padding-b-1';
 c_title.c_title_link_classes = 'lrv-a-unstyle-link lrv-u-color-black';
 c_title.c_title_text = 'Bon Iger: Roseanne is Cancelled and I\'m Like Crying so Hard';
 
-c_tagline.c_tagline_classes = 'lrv-u-color-grey-dark lrv-u-margin-t-00 lrv-a-hidden@mobile-max';
+c_tagline.c_tagline_classes = 'lrv-u-font-size-18 lrv-u-margin-t-00 lrv-a-hidden@mobile-max';
+c_tagline.c_tagline_data_attr = 'data-video-showcase-player-dek';
```

Comment:
I think this is taken care of in the VideoShowcase PR - update this PR to only be adding the span for sponsored badge and the styling